src/globus_cli/commands/flows/list.py

```diff
@@ -16,6 +16,20 @@
     from typing_extensions import Literal
 
 ROLE_TYPES = ("flow_viewer", "flow_starter", "flow_administrator", "flow_owner")
+ORDER_BY_FIELDS = (
+    "id",
+    "scope_string",
+    # These names are the legacy ones in the service:
+    #       "created_by",
+    #       "administered_by",
+    # These are the new names but are not supported at time of writing:
+    #       "flow_owners",
+    #       "flow_administrators",
+    "created_at",
+    "title",
+    "created_at",
```

Comment:
`created_at` appears twice here.

Reply:
No it doesn't! What version are you looking at? Must be at your end because [the file looks perfect to me!](https://github.com/globus/globus-cli/pull/839/commits/027f1d70a37384037f9041274250fc276f8998b0)

(Thanks for spotting this. Probably a whoopsie from refactoring this too many times.)